cloud/tests/conftest.py

```
@@ -0,0 +1,63 @@
+
+class MockSupabaseClient:
+    """Mock Supabase client for tests."""
+
+    def __init__(self):
+        self._responses: dict[str, dict[str, list[Any]]] = defaultdict(lambda: defaultdict(list))
+        self._inserts: list[dict] = []
+
+    def add_response(self, table: str, operation: str, data: list[Any]):
+        self._responses[table][operation] = data
+
+    async def select(self, table: str, columns: str = "*", filters: dict | None = None) -> list[dict]:
```

Ruff (0.15.9)

[warning] Unused method argument: `columns`

(ARG002)